spec/turn_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/card'
+require './lib/turn'
+
+
+RSpec.describe Turn do
+  card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+  turn = Turn.new("Juneau", card)
+
+  it 'exists' do
+    expect(card).to be_a(Card)
+    expect(turn).to be_a(Turn)
+  end
+
+  it 'has a card' do
+    expect(turn.card).to eq(card)
+  end
+
+  it 'has a guess' do
+    expect(turn.guess).to eq("Juneau")
+  end
+
+  it 'can return a boolean for guess' do
```

Comment:
When ever you have a method that can either return a `true` or `false` value, then you want to make sure that you are testing both sides of the condition so that you are sure that the method works in any scenario. The same thing goes for your `feedback` method.